tests/react-directive-spec.js

```diff
@@ -25,6 +25,20 @@ var Hello = React.createClass({
   }
 });
 
+var People = React.createClass({
+  propTypes: {
+    items : React.PropTypes.array
+  },
+
+  render(){
+    var names = this.props.items.map(function(person){
+      return person.fname + ' ' + person.lname;
+    }).join(', ');
```

Comment:
:+1: you could use a arrow function `.map(p => p.fname  + ' ' p.lname).join(',')`